gpytorch/variational/orthogonal_decoupled_variational_strategy.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+import torch
+
+from ..distributions import MultivariateNormal
+from ..utils.memoize import cached
+from ._variational_strategy import _VariationalStrategy
+
+
+class OrthogonalDecoupledVariationalStrategy(_VariationalStrategy):
+    r"""
+    Implements orthogonally decoupled VGPs as defined in `Salimbeni et al. (2018)`_.
+
+    .. _Salimbeni et al. (2018):
+        https://arxiv.org/abs/1809.08820
+    """
+
+    def __init__(self, model, inducing_points, variational_distribution):
+        super().__init__(model, inducing_points, variational_distribution, learn_inducing_locations=True)
+        self.base_variational_strategy = model
+
+    @property
+    @cached(name="prior_distribution_memo")
+    def prior_distribution(self):
+        out = self.model(self.inducing_points)
+        res = MultivariateNormal(out.mean, out.lazy_covariance_matrix.add_jitter())
+        return res
+
+    def forward(self, x, inducing_points, inducing_values, variational_inducing_covar=None):
+        if variational_inducing_covar is not None:
+            raise NotImplementedError("DecoupledVariationalStrategy currently works with DeltaVariationalDistribution")
```

Comment:
This error message?

Reply:
This is true (the variational strategy here is for the mean inducing points). But I can also put that error message in the constructor.

Reply:
Sorry, my comment wasn't very clear. What I meant was:
```suggestion
            raise NotImplementedError("OrthogonalDecoupledVariationalStrategy currently works with DeltaVariationalDistribution")
```

Reply:
I kept the error message there, and added one in the constructor (and changed the class name in the error message to be correct).